packages/client/utils/AtlassianManager.ts

```diff
@@ -600,6 +600,38 @@ export default abstract class AtlassianManager {
     }
   }
 
+  private readonly buildIssueKeyJQL = (queryString: string | null, filteredProjectKeys: string[]) => {
+    if (!queryString) return ""
+    const maybeIssueKeys = queryString.split(/[\s,]+/)
+    if (maybeIssueKeys.length === 0) return ""
```

Comment:
+1 this has a number of eslint errors: double quotes, line length, == instead of ===. 

is the precommit hook broken? we should make sure these get fixed before merging so the next person to open this file doesn't cause a bunch of changes when they save it.